src/pages/products/sgid/location/address-points.astro

```diff
@@ -42,9 +44,6 @@ const page: IPageMetadata = {
     'June 2023 - Beaver, Emery, Davis, Salt Lake, Utah, Washington, and Weber Counties',
     'May 2023 - Cache, Davis, Morgan, Salt Lake, Tooele, Utah, Wasatch, Washington, and Weber Counties',
     'April 2023 - Davis, Morgan, Piute, Salt Lake, Sevier, Summit, Utah, Washington, and Weber Counties',
-    'March 2023 - Box Elder, Cache, Davis, Iron, Salt Lake, Utah, Washington, Weber Counties',
-    'February 2023 - Carbon, Davis, Garfield, Grand, Salt Lake, Tooele, Utah, Washington, Weber Counties',
-    'January 2023 - Box Elder, Davis, Iron, Salt Lake, Utah, Wasatch, Washington, Weber Counties',
```

Comment:
**chore**: we're not removing old updates any longer. Will you add these back please?